pi-manage

```diff
@@ -793,6 +797,95 @@ def create(name, resolver):
     from privacyidea.lib.realm import set_realm
     set_realm(name, [resolver])
 
+# Event interface
+
+@event_manager.command
+def list():
+    """
+    List events
+    """
+    E = EventConfiguration()
+    events = E.events
+    print("{0:7} {4:4} {1:30}\t{2:20}\t{3}".format("Active", "Name", "Module", "Action", "ID"))
+    print(90*"=")
+    for event in events:
+        print("[{0!s:>5}] {4:4} {1:30}\t{2:20}\t{3}".format(event.get("active"),
+                                                 event.get("name")[0:30],
+                                         event.get("handlermodule"),
+                                         event.get("action"),event.get("id"),))
+
+@event_manager.command
+def enable(eid):
+    """
+    enable en event by ID
+    """
+    r = enable_event(eid)
+    print(r)
+
+@event_manager.command
+def disable(eid):
+    """
+    disable an event by ID
+    """
+    r = enable_event(eid, enable=False)
+    print(r)
+
+
+@event_manager.command
+def delete(eid):
+    """
+    delete an event by ID
+    """
+    r = delete_event(eid)
+    print(r)
+
+@event_manager.command
+def e_export(filename=None):
+    """
+    Export all events to a JSON format file
```

Comment:
Hm, we are not really writing JSON because e.g. we use Python ``True`` / ``False`` instead of JSON ``true``/``false`` (same for ``policy p_export``, as I just noticed). I guess we should either convert to JSON, or change the docstring?

Reply:
Then I leave the json comment. The import works fine currently.

Reply:
:+1: